docs/contact.rst

```diff
@@ -6,19 +6,14 @@ Get in touch
 
 .. _get_support_ref:
 
-Getting support
----------------
+Getting support from Learning Equality and from our community
+-------------------------------------------------------------
```

Comment:
just a heads-up that the title is a bit long and will wrap. not sure if this is a concern or not

![image](https://user-images.githubusercontent.com/2367265/41731312-efda689a-7532-11e8-8803-90f560bbd8f9.png)

![image](https://user-images.githubusercontent.com/2367265/41731353-102c4564-7533-11e8-97a3-3eb4547e999c.png)

Reply:
Thoughts @carinediaz @laurenlichtman ? ^^

Reply:
Why can't it be called "Getting Support"?

Reply:
My guess is that @carinediaz wanted to put more emphasis on community support...

Reply:
I think it's okay to leave it at "Getting support" since that would be the objective of the one reading the navigation, then immediately explaining about community, which is also somewhat okay now -- but can of course be optimized in many ways :)

For instance, we could add a "Support others" sort of sub-heading or just a bold text in a paragraph below:

"If you are looking for support, please consider to **support others**. Our common goal is shared, and no one is financed to give you support on an open source project".

Reply:
Any thoughts on the text that I drafted above?

> If you are looking for support, please consider to support others. Your goals are likely overlapping with other users of Kolibri, so you may consider yourself as part of a community of like-minded practitioners!